bert/common/bert.py

```diff
@@ -78,7 +78,7 @@ def tokenize(examples, mode='concat'):
                                     keep_in_memory=True,
                                     remove_columns='text')
 
-    CONFIG['model']['vocab_size'] = len(tokenizer)
+    CONFIG['model']['vocab_size'] = 50304

```

Comment:
is this line necessary?

Reply:
It's used for debug and I forget to remove it. Done.